bolt/test/Inputs/lsda.ldscript

```diff
@@ -0,0 +1,7 @@
+SECTIONS {
+  .text : { *(.text*) }
```

Comment:
lld creates a PF_R PT_LOAD. For best portability, the first section of each segment should be specified. I created #93763